### PR TITLE
[no-ticket] Update DEMO.md

### PR DESCRIPTION
**Pull request by r00ta**
[Insert JIRA Issue number and update link](https://issues.redhat.com/browse/MGDOBR-) 

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: DEMO.md [hunk 1]
```diff
@@ -73,10 +73,12 @@ the response should look like
   "href":"/api/v1/bridges/87508471-ee0f-4f53-b574-da8a61285986",
   "submitted_at":"2021-09-24T11:29:33.086649+0000",
   "status":"AVAILABLE",
-  "endpoint":"/ingress/events/87508471-ee0f-4f53-b574-da8a61285986"
+  "endpoint":"http://ob-87508471-ee0f-4f53-b574-da8a61285986-ob-kekkobar.apps.openbridge-dev.fdvn.p1.openshiftapps.com/"
```
Comment (robpblake) [inline DEMO.md:76]: @r00ta Unrelated to this PR, but does the Ingress support plain HTTP? Pretty sure we should only support HTTPS connections to the ingress.

**Review by robpblake — APPROVED** [review]

File: DEMO.md (hunk 1 above)
Comment (r00ta) [reply DEMO.md:76]: @robpblake yes we should. We still have to configure the cluster properly

Comment (r00ta) [reply DEMO.md:76]: https://issues.redhat.com/browse/MGDOBR-183

Comment (robpblake) [reply DEMO.md:76]: Thanks @r00ta . AFAICR our demo cluster is using LetsEncrypt for SSL certs, but something for us to check.

**Pull request merged**